<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="django_mobile_repair@localhost" uuid="cdee92a8-0036-4a03-9958-cdd02705b7de">
      <driver-ref>postgresql</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
      <jdbc-url>jdbc:postgresql://localhost:5432/django_mobile_repair</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="f33bf391-390a-4b6a-bbfc-26843a56ee11">
      <driver-ref>postgresql</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/project/settings.py</remarks>
      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
      <jdbc-url>jdbc:postgresql://127.0.0.1:5432/django_mobile_repair</jdbc-url>
    </data-source>
  </component>
</project>